### PR TITLE
feat: config tags. remove namespace & account columns. rename  tags->labels

### PR DESCRIPTION
**Pull request by adityathebe**

### CODE REVIEW

File: functions/drop.sql
Comment (adityathebe) [inline functions/drop.sql:1]: droping `namespace` column was not possible as a couple of config views depended on it.

Needed a way to drop those views before we ran the atlas schema. So I put it in here.

File: schema/config.hcl [hunk 1]
```diff
@@ -242,7 +234,7 @@ table "config_items" {
     null = true
     type = text
   }
-  column "tags" {
```
Comment (adityathebe) [inline schema/config.hcl:245]: didn't find any way to rename columns using hcl schemas.

File: query/resource_selector.go [hunk 2]
```diff
@@ -56,7 +56,7 @@ func SearchResources(ctx context.Context, req SearchResourcesRequest) (*SearchRe
 				output.Configs = append(output.Configs, SelectedResource{
 					ID:        items[i].GetID(),
 					Agent:     items[i].AgentID.String(),
-					Labels:    lo.FromPtr(items[i].Tags),
+					Labels:    lo.FromPtr(items[i].Labels),
```
Comment (moshloop) [inline query/resource_selector.go:59]: We should be returning tags here, not labels

File: views/006_config_views.sql [hunk 3]
```diff
@@ -487,7 +520,7 @@ CREATE FUNCTION related_configs_recursive (
     direction TEXT,
     related_id uuid,
     depth INTEGER,
-    tags jsonb,
+    labels jsonb,
```
Comment (moshloop) [inline views/006_config_views.sql:523]: this should be tags

File: views/006_config_views.sql [hunk 4]
```diff
@@ -368,9 +374,36 @@ FOR EACH ROW
...
+    ci.is_pushed, 
+    ci.last_scraped_time, 
+    ci.name, 
+    ci.parent_id, 
+    ci.path, 
+    ci.properties, 
+    ci.scraper_id, 
+    ci.source, 
+    ci.status, 
+    ci.labels,
```
Comment (moshloop) [inline views/006_config_views.sql:404]: need tags here

File: views/006_config_views.sql [hunk 5]
```diff
@@ -168,6 +169,11 @@ CREATE OR REPLACE VIEW config_tags AS
   SELECT d.key, d.value
   FROM configs JOIN json_each_text(tags::json) d ON true GROUP BY d.key, d.value ORDER BY key, value;
 
+-- config_labels
```
Comment (moshloop) [inline views/006_config_views.sql:172]: we probably need a config_tags as well

Comment (adityathebe) [reply views/006_config_views.sql:172]: the previous `config_tags` view also exists.

**Pull request merged**